swift/Workflow/Sources/WorkflowHost.swift

```diff
@@ -65,6 +69,23 @@ public final class WorkflowHost<WorkflowType: Workflow> {
             self?.handle(output: output)
         }
 
+        runLoopObserver = RunLoopObserver(activityStages: .beforeWaiting, runLoopModes: .commonModes) { [weak self] activity in
+            self?.renderIfNeeded()
+        }
+    }
+
+    public func setNeedsRender() {
```

Comment:
Why is this public?

Reply:
🤷‍♀ Maybe you need to poke it?

It should likely start off `internal` or even `private`.

Reply:
Yeah, I don’t think there’s a scenario where the outside world should ever need to force a render (without, say, an `update(workflow:)` call)